lib/entity_store_sequel/postgres_entity_store.rb

```diff
@@ -195,18 +195,23 @@ def get_entities(ids, options={})
     def get_events(criteria)
       return {} if criteria.empty?
 
-      result = {}
+      query = events.where('1=2')
```

Comment:
Ugly hack but Sequel's query building is rather clunky when greater than comparisons are needed